docs/docs/visual-testing-with-storybook.md

```diff
@@ -21,6 +21,130 @@ npx -p @storybook/cli sb init
 
 This command adds a set of boilerplate files for Storybook in your project. However, since this is for a Gatsby project, you need to update the default Storybook configuration a bit so you don't get errors when trying to use Gatsby specific components inside of the stories.
 
+## Storybook version 5.3+
+
+Storybook version 5.3 [brought a major change to how Storybook is configured](https://medium.com/storybookjs/declarative-storybook-configuration-49912f77b78).
+
+When you first install Storybook the only configuration file that will exist is `.storybook/main.js`, which will have the default stories location and default addons.
+
+```js:title=.storybook/main.js
+module.exports = {
+  stories: ["../stories/**/*.stories.js"],
+  addons: ["@storybook/addon-actions", "@storybook/addon-links"],
+}
```

Comment:
@aaronsturm it seems that Storybook introduced a preset when you initialize it. To allow the reader to have the most accurate information, then this code block should be the following:
```javascript
module.exports = {
  stories: ["../stories/**/*.stories.js"],
  addons: ['@storybook/preset-create-react-app','@storybook/addon-actions', '@storybook/addon-links'],
};
```

Reply:
@jonniebigodes - Yes block was the default `.storybook/main.js` when I ran `npx -p @storybook/cli sb init`. I have 3 sections, one is what the user would start with, one is with the changes to introduce, and one is the final product all together. I did this to help bring clarity.

I haven't needed to add `'@storybook/preset-create-react-app'` to make this work. Is it required?

Reply:
the preset is added by default when storybook is initialized as per a recent release. I mentioned it to keep the documentation as accurate as much as possible.